distribution/docs/src/main/jdocs/content/_tables/org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin-table-contents.adoc

```diff
@@ -0,0 +1,35 @@
+:type: documentation
+:status: published
+
+.[[org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin]]Catalog Metacard Ingest Network Plugin
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Criteria
+|criteriaKey
+|String
+|Specifies the criteria for the test of equality; which value will be tested? IP Address? Hostname?
+|remoteAddr
+|true
+
+|Expected Value
+|expectedValue
+|String
+|The value that the criteria must equate to for the attribute overrides to occur.
+|
+|true
+
+|New Attributes
+|newAttributes
+|String"
+|New metacard attributes to apply; if an attribute is specified here, it will be placed on the metacard only if the attribute is not already set. The format should be 'key=value' where 'value' can be a comma-separated list of strings. Whitespace is ignored.
```

Comment:
Correct, but we should probably be more specific. The word _between_ is key. And we have to account for whitespace around `=` so that, when you put it all together, it is clear to the user reading the docs that something like this:
```
name = Billy Bob   ,   Wilma   !   ,   +++,++
```
becomes a setter for attribute `name` using the collection `Billy Bob`, `Wilma   !`, `+++`, and `++` if and only if the attribute `name` was not previously set.